readthedocs/api/v3/tests/responses/projects-builds-detail.json

```diff
@@ -10,6 +10,9 @@
         "project": "https://readthedocs.org/api/v3/projects/project/",
         "version": "https://readthedocs.org/api/v3/projects/project/versions/v1.0/"
     },
+    "urls": {
+        "build": "https://readthedocs.org/projects/project/builds/1/"
+    },
```

Comment:
What about showing the same URLs we are listing in `_links`?

* `_self`
* `project`
* `version`

I think it makes sense to keep that matching between the two. `_links` refers to API URLs and `urls` refer to user/dashboard URLs. Both with the same naming.

Reply:
@humitos  Yeah! That makes sense :+1: . Updated the PR